README.md

```diff
@@ -132,6 +132,8 @@ Once finished, it will copy the HTML file to the out folder.
 There are some limitations:
 - Modules are not supported (Basis and Ammo)
 - Texture compression formats are not supported
+- Spine runtime is not supported
```

Comment:
Spine is not supported as it relies on the URL to extract a filename which is not possible with a URL that is Base64 encoded.